CovidTracker/Pages/Covid19CasesAndDeaths.cshtml.cs

```diff
@@ -21,8 +21,20 @@ public void OnGet(string query)
             InitStateDropdown();
             using (var webClient = new WebClient()) 
             {
-                string jsonString = webClient.DownloadString("https://data.cdc.gov/resource/9mfq-cb36.json");
-                var covid19CasesAndDeaths = CasesAndDeaths.Covid19CasesAndDeaths.FromJson(jsonString);
+                string covid_casesndeath_data=string.Empty;
+
+                try
+                {
+                    covid_casesndeath_data = webClient.DownloadString("https://data.cdc.gov/resource/9mfq-cb36.json");
+                }
+                catch (Exception e)
+                {
+                    Console.WriteLine("Exception while calling API - Covid Vaccines", e);
+
+                }
```

Comment:
Logging is a good idea, but what should you do with the exception at this point?  Re-throw it?  If you just log it and bury it, the program will continue to execute after the exception occurs, as if nothing went wrong.  That is worse than the default behavior: terminate the method when an exception occurs.

Use proper logging instead of Console.WriteLine().
- Logging has the concept of priority (error, debug, warn, etc.), which Console.WriteLine() doesn't have.
- No one will see Console.WriteLine(), as it goes to console. 
- Log messages can be aggregated by several commercial tools.  Not so easy with Console.WriteLine().